apis/container/v1beta2/zz_cluster_types.go

```diff
@@ -13,6 +13,40 @@ import (
 	v1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
 )
 
+type AdditionalNodeNetworkConfigsInitParameters struct {
```

Comment:
Should we backport this and other empty structs from the changes in the API of the `Cluster.container` resource?

Reply:
We discussed this topic off-channel. Here's the summary: Automatic backporting script begins from "root types", such as `ClusterInitParameters`, `ClusterParameters`, and `ClusterObservation`, and backports only the types that descend from their type hierarchy. Doing so was necessary to handle some edge cases, which caused the “same” struct to have a different name in newer API versions. Because these empty structs, mentioned in the review comment, are not used in anywhere, they are not backported. I argue that we shouldn't backport them manually, simply because doing so won't have any benefits.